src/fx/Storage.js

```diff
@@ -0,0 +1,102 @@
+import { assign } from "../utils.js"
+
+function writeToStorage(props) {
+  var storage = window[(props.storage || "session") + "Storage"]
```

Comment:
Is there a particular reason you're defaulting to `sessionStorage`? I see `localStorage` being used more frequently due to the persistence behavior it has.

Reply:
My thinking was that if you want to save something that will persist across browser session that you'd have to be explicit about it. I'm not wed to using `sessionStorage` by default.

I was also toying with the idea of using `props.expires` (boolean) instead of `props.storage` (either "session" or "local"). But it felt a bit weird having to specify `expires` in the `ReadFromStorage` effect

Reply:
I prefer `props.storage` but I would also like the default to be `localStorage`.